xml/bp_troubleshooting_objectgateway.xml

```diff
@@ -61,7 +61,7 @@
     </para>
  </sect1>
  <sect1 xml:id="bp-troubleshooting-crashed-ragosgw">
-   <title>Crashed &rgw; Process</title>
+   <title>Crashed &ogw; Process</title>
    <para>
      If the radosgw process dies, you will normally see a 500 error from
      the web server. In that situation, restarting radosgw will restore service:
```

Comment:
It looks like these were not entities, but maybe should have been? (I am not sure.) Want to make them entities or is it fine as-is?

Reply:
I noted there was a lot of this through the guides too. It would take a considerable effort to go through and change. I think we stick to the premise of changing the incorrect entity for now, and as we go through we can change where applicable :)

Reply:
Sounds legit. In which case, LGTM, go for it.